wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WTable.java

```diff
@@ -511,6 +511,25 @@ public void setSeparatorType(final SeparatorType separatorType) {
 	}
 
 	/**
+	 * Sets whether the de/selection of a row with sub rows should de/select the sub rows. This is a client-side only
+	 * feature.
+	 *
+	 * @param toggleSubRowSelection true to turn on this feature.
+	 */
+	public void setToggleSubRowSelection(boolean toggleSubRowSelection) {
+		getOrCreateComponentModel().toggleSubRowSelection = toggleSubRowSelection;
+	}
+
+	/**
+	 * Indicates whether de/selecting a row with sub row(s) will de/select the sub row(s).
+	 *
+	 * @return true if this feature is enabled.
+	 */
+	public boolean getToggleSubRowSelection() {

```

Comment:
As it is boolean, the naming convention should be "isToggleSubRowSelection" on the getter.